### ISSUE TITLE
Share folder with images fullscreen bug

### ISSUE THREAD

**Issue by gamecraft45** [pydio/pydio-core#1168]
I tried in Chrome and Opera it look same
![image](https://cloud.githubusercontent.com/assets/7279856/14762978/874f0c62-0988-11e6-8421-b4983c62939a.png)
![image](https://cloud.githubusercontent.com/assets/7279856/14763008/552708a6-0989-11e6-98ec-a1a18395f394.png)

**Comment by dosenbier** [comment]
Same problem with fullscreen not working in shared links.

IE11: Not working at all (white screen)
Firefox/Chrome/Edge: Adding a white border (as see on the screenshot). 
Tested on Win7 and Windows 10. 

I've tried a complete new install from version Pydio-Core 6.4.1 - 2016-04-11, but no difference.

Shared link -Browser view:
![browser_view](https://cloud.githubusercontent.com/assets/19623800/15627686/14068076-24ec-11e6-8312-d392fdbcf39d.jpg)

Shared link - Fullscreen
![fullscreen_view](https://cloud.githubusercontent.com/assets/19623800/15627688/176c8df0-24ec-11e6-9491-034f1a230504.jpg)
